src/views/router_views/individualPscList/individualPscList.njk

```diff
@@ -1,35 +1,136 @@
+    <p class="govuk-body">{{ i18n.ind_psc_list_can_verify_description }}<b>{{ companyName }}</b>{{ i18n.ind_psc_list_can_verify_description2 }}</p>
+    
+{% for psc in pscDetails | sort(attribute = 'pscName') %}
+
+{% set pscStatusText %}
+  <strong class="govuk-tag govuk-tag--red"> {{ i18n.ind_psc_list_status_unverified }} </strong> 
+{% endset %}
+
+    {{ govukSummaryList({
+    card: {
+        title: {
+        text: psc.pscName
+    },
+    actions: {
+        items: [
+            {
+            href: [nextPageUrl+psc.pscId ],
+            text: i18n.ind_psc_list_summary_card_verify_link,
+            visuallyHiddenText: i18n.ind_psc_list_summary_card_verify_link
+            }
+        ]
+    }
+  },
+  rows: [
+    {
+      key: {
+        text: i18n.ind_psc_list_summary_card_key1
+      },
+      value: {
+        text: psc.pscDob
+      }
+    },
+    {
+      key: {
+        text: i18n.ind_psc_list_summary_card_key2
+      },
+      value: {
+        text: psc.pscVerificationDeadlineDate
+      }
+    },
+    {
+      key: {
+        text: i18n.ind_psc_list_summary_card_key3
+      },
+      value: {
+        html: pscStatusText
+      }
+    }
+  ]
+}) }}
+
+{% endfor %}
+
+<h2 class="govuk-heading-m">{{ i18n.ind_psc_list_cannot_verify_heading }}</h2>
+<p class="govuk-body">{{ i18n.ind_psc_list_cannot_verify_description }}<b>{{ companyName }}</b>
+{{ i18n.ind_psc_list_cannot_verify_description2 }}<b>{{ confirmationStatementDate }}</b>{{ i18n.ind_psc_list_cannot_verify_description3 }}</p>
+
+{{ govukDetails({
+    summaryText: i18n.ind_psc_list_cannot_verify_details_main_link,
+    text: i18n.ind_psc_list_cannot_verify_details_description,
+    attributes: {
+        "data-event-id": "why-do-pscs-need-to-wait-dropdown"
+    }
+}) }}
+
+{% set unverifiablePscStatusText %}
+  <strong class="govuk-tag govuk-tag--red"> {{ i18n.ind_psc_list_status_unverified }} </strong> 
+{% endset %}
+
+{{ govukSummaryList({
+    card: {
+        title: {
+        text: "[Name of the psc]"
+    }
+  },
+  rows: [
+    {
+      key: {
+        text: i18n.ind_psc_list_summary_card_key1
+      },
+      value: {
+        html: "[partial_dob]"
+      }
+    },
+    {
+      key: {
+        text: i18n.ind_psc_list_summary_card_key2
+      },
+      value: {
+        html: "[verification_deadline]"
+      }
+    },
+    {
+      key: {
+        text: i18n.ind_psc_list_summary_card_key3
+      },
+      value: {
+        html: unverifiablePscStatusText
+      }
+    }
+  ]
+}) }}
+
+{% set pscNotFoundDetails %}
+        <p class="govuk-body">{{ i18n.ind_psc_list_not_found_details_description }}</p>
+        <p class="govuk-body">{{ i18n.ind_psc_list_not_found_details_description2 }}</p>
+        <p class="govuk-body">{{ i18n.ind_psc_list_not_found_details_description3 }}{{ dsrEmailAddress }}
+            {{ i18n.ind_psc_list_not_found_details_description4 }}{{ dsrPhoneNumber }}
+            {{ i18n.ind_psc_list_not_found_details_description5 }}</p>
```

Comment:
Just a note on something I've seen before, but not found a solution(!): extra white space follows a variable inserted within the JSON text e.g. after `{{ dsrPhoneNumber }}`.